telegram/_bot.py

```diff
@@ -930,6 +962,46 @@ async def delete_message(
             api_kwargs=api_kwargs,
         )
 
+    @_log
+    async def delete_messages(
+        self,
+        chat_id: Union[int, str],
+        message_ids: Sequence[int],
+        *,
+        read_timeout: ODVInput[float] = DEFAULT_NONE,
+        write_timeout: ODVInput[float] = DEFAULT_NONE,
+        connect_timeout: ODVInput[float] = DEFAULT_NONE,
+        pool_timeout: ODVInput[float] = DEFAULT_NONE,
+        api_kwargs: Optional[JSONDict] = None,
+    ) -> bool:
+        """
+        Use this method to delete multiple messages simultaneously. If some of the specified
+        messages can't be found, they are skipped.
+
+        .. versionadded:: NEXT.VERSION
+
+        Args:
+            chat_id (:obj:`int` | :obj:`str`): |chat_id_channel|
+            message_ids (Sequence[:obj:`int`]): Identifiers of 1-100 messages to delete.
```

Comment:
We'll need new constants for these limits (1 & 100). Maybe we have a `constants.BulkRequestLimit` since the limits for `(delete|copy|forward)_messages` are all the same …